content/docs/setup-and-administration/configuring-app-registrations-for-non-interactive-use/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Configuring App Registrations for Non-Interactive Use"
+description: "Creating App Registrations for RKVST"
+lead: "Creating App Registrations for RKVST"
+date: 2021-06-16T11:12:25+01:00
+lastmod: 2021-06-16T11:12:25+01:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "setup-and-administration"
+weight: 12
+toc: true
+---
+
+Non-interactive access to the RKVST platform is managed by creating `App Registrations` either via the Manage RKVST Interface in the UI or by using the App Registrations API directly.
+
+`App Registrations` are Authentication Profiles with a `CLIENT ID` and `SECRET` that can then be used to generate JWT Tokens for authenticating to RKVST API Endpoints.
+
+## Creating an App Registration
+
+To enable non-interactive access to RKVST you **must** create your first `App Registration` in the RKVST UI and **only** as a Root User.
+
+### Using the RKVST UI (Required for First-Time Setup)
+
+1. As a Root User visit the APP REGISTRATIONS tab in the `Manage RKVST` Interface
+2. Click CREATE APP REGISTRATION.
+3. Enter any display name you like.
+a. Optional - using the ADD CUSTOM CLAIM button Add any extra claims you require in your access token.
+4. Click CREATE APP REGISTRATION. The response will include the CLIENTID and SECRET required by the archivist token endpoint.
+
+{{< caution >}}
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+{{< /caution >}}
+
+5. Now you have created your App Registration, follow the steps further below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
+
+### Using the App Registrations API
+
+The following assumes you already have at least one `App Registration` configured and that you are comfortable generating tokens and using the RKVST API.
+
+If you do not yet have an App Registration configured please follow [the first-time setup guide](./#creating-an-app-registration) to get started.
```

Comment:
This links to the section we're already in, I think?

Reply:
It links to the top level of the section as the (First Time setup) is a little awkward, I can correct it probably though

Reply:
Fixed, I was just too lazy to type out the (first-time-setup) bit at 2am now it links to the right place